types/inflected/index.d.ts

```diff
@@ -1,43 +1,59 @@
-// Type definitions for inflected 1.1.6
+// Type definitions for inflected 2.1.0
```

Comment:
```suggestion
// Type definitions for inflected 2.1
```
just MAJ.MIN:
https://github.com/DefinitelyTyped/DefinitelyTyped#how-do-definitely-typed-package-versions-relate-to-versions-of-the-corresponding-library